Add more information (#1399)
ballerina-platform/ballerina-www: website/two-column-pages/docs/learn/quick-tour.md

> **Note**: You need to have VS Code installed to try this. You can download it from [https://code.visualstudio.com/Download](https://code.visualstudio.com/Download).

> Open your service in VS Code. You can use the following command to do this on Linux or OSX. Replace '/<folder_path>/' by the actual folder path where the Ballerina project was initialized.

Reviewer: ```suggestion
Open your service in VS Code. You can use the following command to do this on Linux or OSX. Replace '/<folder_path>/' with the actual folder path in which the Ballerina project was initialized.
```
